Wikipedia/Categories/NSAttributedString+WMFModifyParagraphs.h

```diff
@@ -0,0 +1,20 @@
+//  Created by Monte Hurd on 8/17/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import <Foundation/Foundation.h>
+
+@interface NSAttributedString (WMFModifyParagraphs)
+
+/**
+ *  Sometimes you need to adjust all paragraph styles without destroying existing paragraph style properties.
```

Comment:
comment styling nit-pick: doc comments should have 1 brief line describing the method's purpose, then if needed, a longer paragraph with details & usage examples. For example:

```
/**
 * A method that does a thing
 * 
 * Useful for doing stuff, with things. 
 *
 ... params & return comments, etc.
```